libs/blocks/src/lib/hero/content-bottom/index.tsx

```diff
@@ -12,7 +12,7 @@ const ContentBottom = ({
       className={clsx(
         'md:py-10 md:px-6',
         'lg:py-4000',
-        'w-full flex flex-col py-10 px-4  bg-background-primary-container gap-10',
+        'h-screen w-full flex flex-col py-10 px-4 bg-background-primary-container gap-10',
```

Comment:
Can we remove `h-screen` we are not controlling the height and instead let the content flow instead.